apps/AmplifyDemo/src/components/MeetingForm.tsx

```diff
@@ -0,0 +1,104 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT-0
+
+import React, { ChangeEvent, FC, FormEvent, useState } from 'react';
+
+import {
+  Flex,
+  FormField,
+  Input,
+  PrimaryButton,
+  useMeetingManager,
+} from 'amazon-chime-sdk-component-library-react';
+import { addAttendeeToDB, addMeetingToDB, createMeeting, getAttendeeFromDB, getMeetingFromDB, joinMeeting } from '../utils/api';
```

Comment:
Recommend formatting them in multiple lines like the above one (`amazon-chime-sdk-component-library-react`).